modules/nf-core/propr/propr/tests/main.nf.test

```diff
@@ -0,0 +1,163 @@
+nextflow_process {
+
+    name "Test Process PROPR_PROPR"
+    script "../main.nf"
+    process "PROPR_PROPR"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "propr"
+    tag "propr/propr"
+
+    test("Test propr/propr using default options") {
+
+        tag "default"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file(params.test_data['mus_musculus']['genome']['rnaseq_matrix'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+        
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.matrix).match("Test propr/propr using default options - matrix") },
+                { assert snapshot(process.out.versions).match("versions") }
```

Comment:
missing fdr check here

Reply:
hey there! fdr is an optional output actually. Here in the default test, I did not ask to compute fdr.

Reply:
Ahh, my bad, thanks for explanation